### ISSUE TITLE
Missing Go Tool

### ISSUE THREAD

**Issue by jeffreyjw** [joefitzgerald/go-config#30]
The editor prompts me to install gocode and cover tool, however I have already installed them manually via go get. When i try to install it again in the editor the following message appears:

`The go tool is required to perform a get. Please ensure you have a go runtime installed: http://golang.org.`

So I'm pretty sure there's something wrong with the detection, but I can't get any info what is being actually detected in the system: in other issues I've seen you suggested to look into `Packages > Go-plus > ...` and there should be an appropriate option to get this info (can't remember the exact name of the option right now - but when I looked it up, I didn't have it). I don't even have the `go-plus` menu under `Packages`, only go (maybe this is irrelevant). I attached a screenshot to show what I have there.
![goplus](https://cloud.githubusercontent.com/assets/3730812/14412259/8e734ffa-ff5e-11e5-8d2c-73a113713285.png)

I also found no option to check what has been detected in `go-config` and `go-plus` packages `Settings`.

To sum up: is there a way I can see what `go-config` actually detected?

**Comment by joefitzgerald** [comment]
Launch atom from your terminal if you are on atom < 1.7.0 or on Linux.

**Comment by jeffreyjw** [comment]
Yeah already tried it and the result is the same unfortunately :( 
Sorry for forgeting to write about my configuration - my OS is Mint Rafaela (practically an Ubuntu 14.04) and Atom 1.6.2

**Comment by joefitzgerald** [comment]
Try the suggestions in this: https://github.com/joefitzgerald/go-plus/issues/386#issuecomment-199359955. Remember, you can press `ctrl-shift-i` to get to the chromium debugger, from which you can go to the console and write `process.env` to debug.

**Comment by jeffreyjw** [comment]
I found the solution: I had  long time ago installed `go` using `apt`, and this binary was the source of problem - still don't know why exactly, but removing it and having a once again clean install from tarball did the trick.

If this information have any significant matter for you for the future: `GOPATH` and `PATH` were OK inside `process.env` before I removed this package from `apt`.

Thank you for very quick responses and have a good day :)

**Comment by joefitzgerald** [comment]
Glad to hear you got it resolved!

**Comment by xh3b4sd** [comment]
I had the same issue on OSX where I installed go with brew. Turned out it fixed it for me when getting rid of the `GOROOT` env var. See also http://stackoverflow.com/a/35823325/3479043 and the linked blog post of Dave Cheney.
